src/controller.js

```diff
@@ -25,6 +25,15 @@ export default class Controller {
     _element(query).classList.remove(classList)
   }
 
+  toggleClass(query, classList) {
+    const element = _element(query)
+
+    if (element.classList.contains(classList))
```

Comment:
if object with the class exist
```suggestion
    if find(`${query}.${className}`)
```
```
    removeClass(query, className)
  else
    addClass(query, className)
```